frontend/ui/widget/bookmapwidget.lua

```diff
@@ -1292,14 +1307,28 @@ Under the pages, these indicators may be shown:
 ▒ highlighted text
  highlighted text with notes
  bookmarked page
-▢ focused page when coming from Pages browser
+▢ focused page when coming from Pages browser]])
 
-On a newly opened book, the book map will start in grid mode showing all chapter levels, fitting on a single screen, to give the best initial overview of the book's content.]]),
-    })
+    if self.overview_mode then
+        text = text .. "\n\n" .. _([[
+In overview mode, the book map is always in grid mode and ensured to fit on a single screen. Chapter levels can be changed for the most confortable overview.]])
```

Comment:
@Frenzie : no comment about this added sentence wording ?
(The rest is just splitted previous text :/ for reuse in the 2 cases.)

Reply:
Actually I hadn't noticed that there was some added. There's a French typo in comfortable.

And maybe "made to fit" would sound a touch better than "ensured to fit"?